virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-shared-volumes-for-vms"]
+= Configuring shared volumes for virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-shared-volumes-for-vms
+
+toc::[]
+
+
+You can configure shared disks to allow multiple virtual machines (VMs) to share the same underlying storage. A shared disk's volume must be block mode.
+
+You configure disk sharing by exposing the storage in either of the following ways:
+
+* As an ordinary virtual machine disk.
```

Comment:
I'm not sure that it is a good idea to begin sentence fragments with "As a....". If you can find a way to rewrite this,  it would be more readable.

Reply:
Done